aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,285 @@
+
+Resource files using ``@inject IViewLocalizer`` in Razor views follow a similar pattern. The resource file for a view can be named using either dot naming or file naming. Razor view resource files mimic the path of their associated view file. Assuming we set the ``ResourcesPath`` to "Resources", the French resource file associated with the *Views/Book/About.cshtml* view could be either of the following:
+
+- Resources/Views/Home/About.fr.resx
+- Resources/Views.Home.About.fr.resx
+
+If you don't use the ``ResourcesPath`` option, the *.resx* file for a view would be located in the same folder as the view.
+
+Generating resource files with Visual Studio
+-----------------------------------------------
+
+If you create a resource file in Visual Studio without a culture in the file name (for example, *MyStrings.resx*), Visual Studio will create a C# class with a property for each string.  That's usually not what you want with ASP.NET Core; you typically don't have a default resx resource file (A resx file without the culture name). We suggest you create the .resx file with a culture name (for example *MyStrings.fr.resx*). When you create a .resx file with a culture name, Visual Studio will not generate the class file.
+
+If you remove the ".fr" culture designator AND you have the culture set to French (via cookie or other mechanism), the default resource file is read and strings are localized. This is expected behavior. \*.resx designates a default or fallback resource, so that when nothing meets your requested culture you're served \*.resx. If you want to just return the key when missing a resource for the requested culture you must not have a default resource file.
+
+Globalization and Localization in ASP.NET Core
+-------------------------------------------------------
+
+Internationalization involves `Globalization <https://msdn.microsoft.com/en-us/library/aa292081(v=vs.71).aspx>`__ and `Localization <https://msdn.microsoft.com/en-us/library/aa292137(v=vs.71).aspx>`__. Globalization is the process of designing applications that support different cultures. Globalization adds support for input, display, and output of a defined set of language scripts that relate to specific geographic areas. 
```

Comment:
This is some good introductory text that should probably be at the beginning of this article.